rest/src/main/java/fi/otavanopisto/pyramus/rest/StudentRESTService.java

```diff
@@ -3535,5 +3543,60 @@ private Status checkStudent(Student student) {
 
     return Status.OK;
   }
+  
+  @Path("/students/{STUDENTID:[0-9]*}/subjectChoices")
+  @GET
+  @RESTPermit(StudentPermissions.LIST_STUDENT_SUBJECT_CHOICES)
```

Comment:
Does the student need to be able to load their own choices? If so, this needs to move to an inline check where the ownership is checked separately.